ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -1173,21 +1181,23 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
                   You must select a billing account</div>}
                 {errors.primaryPurpose && <div> You must choose at least one primary research
                   purpose (Question 1)</div>}
+                {errors.diseaseOfFocus && <div> You must specify a disease of focus and it should be at most 80 characters</div>}
                 {errors.otherPrimaryPurpose && <div> Other primary purpose should be of at most 500 characters</div>}
                 {errors.anticipatedFindings && <div> Answer for <i>What are the anticipated findings
-                  from the study? (Question # 2.1)</i> cannot be empty</div>}
```

Comment:
We had inconsistent numbering here, so I removed the number symbols where it wasn't consistent.